sound/soc/sof/imx/Kconfig

```diff
@@ -37,6 +38,7 @@ config SND_SOC_SOF_IMX8
 config SND_SOC_SOF_IMX8M_SUPPORT
 	bool "SOF support for i.MX8M"
 	depends on IMX_DSP=y || IMX_DSP=SND_SOC_SOF_OF
+	select SND_SOC_SOF_XTENSA
```

Comment:
@IulianOlaru249  looking at similar Intel change, SND_SOC_SOF_XTENSA should be under SND_SOC_SOF_IMX8 option:

Look at this contextual diff:

```

 
 config SND_SOC_SOF_IMX8
        tristate
+       select SND_SOC_SOF_XTENSA
        help

```